determined-ai/determined
# feat: expose `externalExperimentId` and `externalTrialId`

## Description
[WEB-1632]




## Test Plan
- Check if `externalExperimentId` and `externalTrialId` are in static columns, and these are hidden by default in the new experiment list
- Check if  `externalExperimentId` and `externalTrialId` are filterable and sortable
- Check if `externalTrialId` is the `best_trial_id`




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1632]: https://hpe-aiatscale.atlassian.net/browse/WEB-1632?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
